data/pb/ws-services-calculation/meterStatus.json

```diff
@@ -0,0 +1,36 @@
+{
+  "tenantId": "pb",
+  "moduleName": "ws-services-calculation",
+  "meterStatus": [
+    {
+      "name": "Working",
+      "code": "WORKING",
+      "active": true
+    },
+    {
+      "name": "Locked",
+      "code": "LOCKED",
+      "active": true
+    },
+    {
+      "name": "Breakdown",
+      "code": "BREAKDOWN",
+      "active": true
+    },
+    {
+      "name": "No-meter",
+      "code": "NOMETER",
```

Comment:
Use NO_METER if it makes sense in this context?

Reply:
ok